### ISSUE TITLE
[BUG] New Chat Message Timestamp feature doesn't save new output

### ISSUE THREAD

**Issue by GambetTV** [vigoren/foundryvtt-simple-calendar#514]
This is v10 of Foundry, but I have your latest module working correctly for outputting the timestamps of the calendar into chat. Thank you for adding that feature! However, when saving the chat log, it seems to use IRL timestamps. Is there a way to get the .txt file to output with game time too?

**Comment by vigoren** [comment]
Confirmed the game time is not apart of the chat export. Foundry does not have a nice way to update or change the contents of that export. 
That said I did find a way to do it. It will still include the real life time but has a new line that contains the game time, example of the output would look like:

```
---------------------------
[7/6/2023, 4:32:25 PM] Gamemaster
{Game Time: Sep 02, 2022 00:02
This is a great chat message!
---------------------------
```

The fix will be in the next release (hopefully in a few days)

**Comment by vigoren** [comment]
This has been fixed in version 2.3.6!
